platform/windows.ts

```diff
@@ -1,111 +1,99 @@
-import { Buffer, copy } from "../deps.ts";
-import { decode, encode, writeTmp } from "./helper.ts";
+import { decode } from "./helper.ts";
 
-export async function read_text(): Promise<string> {
-  const p = Deno.run({
-    cmd: ["powershell", "-noprofile", "-command", "Get-Clipboard"],
+export async function readText(): Promise<string> {
+  const cmd = new Deno.Command("powershell", {
+    args: ["-noprofile", "-command", "Get-Clipboard"],
+    stdin: "null",
     stdout: "piped",
     stderr: "piped",
   });
-
-  try {
-    const status = await p.status();
-    if (!status.success) {
-      p.stdout.close();
-      const cause = decode(await p.stderrOutput());
-      throw new Error(
-        `cannot read text: exit code: ${status.code}, error: ${cause}`,
-      );
-    }
-    p.stderr.close();
-    return decode(await p.output()).replace(/\r?\n/, "");
-  } finally {
-    p.close();
+  const { success, code, stderr, stdout } = await cmd.output();
+  if (!success) {
+    const cause = decode(stderr);
+    throw new Error(
+      `failed to read text: exit code: ${code}, error: ${cause}`,
+    );
   }
+  // XXX:
+  // I don't think it's a good idea to remove a newline...
+  // Shall we remove only the trailing one?
```

Comment:
Does that mean there is a problem if I remove the newline?
If so, please let me know what kind of problem you can think of.

Reply:
I'm not familiar with `Get-Clipboard` command of PowerShell but if user had copied `Hello\r\nWorld`, the output of the command will be `Hello\r\nWorld\r\n` and the output of this function will be `HelloWorld\r\n` instead of `Hello\r\nWorld` right?

Reply:
Ah, understood.
Could you please fix it?(remove the trailing)

Reply:
Done. Please re-review @skanehira